ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -23,288 +22,246 @@ module('Acceptance | sync | overview', function (hooks) {
   hooks.beforeEach(async function () {
     this.version = this.owner.lookup('service:version');
     this.permissions = this.owner.lookup('service:permissions');
+    this.version.type = 'enterprise';
   });
 
-  module('ent', function (hooks) {
```

Comment:
this file looks like a LOT of changes, but it's mostly 
1. replace redirect assertions with checking that link hide/show appropriately
2. removing super nested modules to make the tests easier to follow/edit